FEXCore/Source/Interface/Core/OpcodeDispatcher.cpp

```diff
@@ -3593,8 +3593,18 @@ void OpDispatchBuilder::SGDTOp(OpcodeArgs) {
 
 void OpDispatchBuilder::RDTSCOp(OpcodeArgs) {
   auto Counter = _CycleCounter();
-  auto CounterLow = _Bfe(OpSize::i64Bit, 32, 0, Counter);
-  auto CounterHigh = _Bfe(OpSize::i64Bit, 32, 32, Counter);
+  OrderedNode *CounterLow{};
```

Comment:
Instead of duplicating, pull out a helper CycleCounter() which does _CycleCounter + math and outputs two OrderedNodes for low/high

Reply:
Implementation merged in to a helper.